File: JS/URI/1045.js
var input = require('fs').readFileSync('stdin', 'utf8');

var lines = input.split('\n');

var line = lines.shift().split(" ");

var a = parseFloat(line[0])
var b = parseFloat(line[1])
var c = parseFloat(line[2])

line.sort((a, b) => b - a);
var a = parseFloat(line[0])
var b = parseFloat(line[1])
var c = parseFloat(line[2])

if(a>=b+c )
    {
        console.log("NAO FORMA TRIANGULO");
    }
else if (a*a == b*b+c*c)
    {
        console.log("TRIANGULO RETANGULO");
    }
else if (a*a > b*b+c*c)
    {
        console.log("TRIANGULO OBTUSANGULO");
    }
else if(a*a < b*b+c*c)
    {
        console.log("TRIANGULO ACUTANGULO");
    }
if (a == b && a==c)
    {
        console.log("TRIANGULO EQUILATERO");
    }
if(a==b && a!=c || a==c && a!==b || b==c && b!=a)
    {
        console.log("TRIANGULO ISOSCELES");
    }

File: C/URI/1045.c
#include <stdio.h>
 
int main() {
 
  double a, b, c, aux;
	scanf("%lf %lf %lf", &a, &b, &c);
	  if(a <= b && b <= c)
	    {
		aux = c;
		c = a;
		a = aux;
	    }
	  else if(a <= c && c <= b)
	  {
		aux = b;
		b = c;
		c = a;
	 	a = aux;
	  }
	  else if(b <= c && c <= a)
	  {
		aux = c;
		c = b;
		b = aux;
	  }
	  else if(c <= a && a <= b)
	  {
		aux = b;
		b = a;
		a = aux;
	  }
	  else if(b <= a && a <= c)
	  {
		aux = c;
		c = b;
		b = a;
		a = aux;
	  }
	if(a >= (b + c))
	  printf("NAO FORMA TRIANGULO\n");
	else if((a * a) == ((b * b)+(c * c)))
	  printf("TRIANGULO RETANGULO\n");
	else if((a * a) > ((b * b)+(c * c)))
	  printf("TRIANGULO OBTUSANGULO\n");
	else if((a * a) < ((b * b)+(c * c)))
	  printf("TRIANGULO ACUTANGULO\n");
	 if(a == b && a == c)
	  printf("TRIANGULO EQUILATERO\n");
	 if(a == b && b != c || c != a && c == b)
	  printf("TRIANGULO ISOSCELES\n");


    return 0;
}